addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,241 @@
 
 <template id="index" name="Find Resellers">
     <t t-call="website_crm_partner_assign.layout">
-        <div class="col-lg-12">
-            <h1 class="text-center">
-                Looking For a Local Store?
-            </h1><h2 class="text-center text-muted">
-                Contact a reseller
-            </h2>
-        </div>
-
-        <div class="col-lg-3 mb32" id="partner_left">
-
-            <ul id="reseller_grades" class="nav nav-pills flex-column mt16">
-                <li class="nav-header nav-item"><h3>Filter by Level</h3></li>
-                <t t-foreach="grades" t-as="grade">
-                    <li class="nav-item">
+        <div class="o_wcrm_filters_top d-flex d-print-none align-items-center justify-content-end flex-wrap gap-2 w-100">
+            <h4 class="my-0 me-auto pe-sm-4">
+                Find a reseller <t t-foreach="countries" t-as="country"><span t-if="country['active'] and country['country_id'][0] != 0 and country['country_id']">in <t t-out="country['country_id'][1]"/></span></t>
```

Comment:
I tried by using the exact same line you proposed (by closing the `next()`function after the loop : `t-value="next(country['country_id'][1] for country in countries) if country['active'] and country['country_id'][0] != 0"` and it returns an internal error. I also tried without the `next()` and it doesn't return an internal but it displays like this :

![Screenshot 2023-05-03 at 13 41 30](https://user-images.githubusercontent.com/110090660/235906196-5e406df6-755c-43f6-8f5a-1607604b2ed0.png)

Do you have an idea on how I can fix that easily ? Otherwise I will just keep what I did first and I add the comment you suggested to add 

Thanks !

Reply:
Should have been `next(country['country_id'][1] for country in countries if country['active'] and country['country_id'][0] != 0)`, bracket ended too soon.

But you can keep the other one it's fine too, was just a nitpick it doesn't have much impact ^^'